server/order/match.go

```diff
@@ -17,3 +17,10 @@ type Match struct {
 	Rates   []uint64
 	Total   uint64
 }
+
+type OrderPair struct {
```

Comment:
Will be removed in favor of the new `Match` type defined in PR https://github.com/decred/dcrdex/pull/39